environment.yml

```diff
@@ -0,0 +1,20 @@
+name: symbiflow_arch_def_base
+channels:
+  - defaults
+  - symbiflow
+dependencies:
```

Comment:
Agreed.  I also added make and ninja.  We are techinically missing a compiler toolchain, but I think we should *not* add that, and instead move the binary compilation into conda packages.  This may have already been done, but I want to leave that change for a follow-up PR.  There is already enough churn here.

Reply:
Yeah that sounds reasonable.

Reply:
FYI, I removed ninja because a system ninja and the conda ninja interact poorly.  As a follow up, we should figure out a good way to have people enter the conda environment.  My thinking was to put shell scripts in <root>/build like `activate` and `deactivate`.